variables.tf

```
@@ -104,6 +104,12 @@ variable "load_balancer_type" {
   default     = "application"
 }
 
+variable "enforce_security_group_inbound_rules_on_private_link_traffic" {
+  description = "Indicates whether inbound security group rules are enforced for traffic originating from a PrivateLink. Only valid for Load Balancers of type network. The possible values are on and off."
+  type        = string
+  default     = "off"
```

This is an option value so we should default to `null`
```suggestion
  default     = null
```